manager-rs/dauth-service/src/manager.rs

```diff
@@ -183,27 +183,30 @@ pub async fn confirm_auth_vector(
                 tracing::info!("Auth started from backup network");
 
                 let mut key_shares = Vec::with_capacity(backup_network_ids.len());
+                let mut responses = Vec::with_capacity(backup_network_ids.len());
 
                 for backup_network_id in backup_network_ids {
                     let (backup_address, _) =
                         clients::directory::lookup_network(context.clone(), &backup_network_id)
                             .await?;
-                    match clients::backup_network::get_key_share(
+
+                    responses.push(tokio::spawn(clients::backup_network::get_key_share(
                         context.clone(),
-                        &xres_star_hash,
-                        &res_star,
-                        &backup_address,
-                    )
-                    .await
+                        xres_star_hash.clone(),
+                        res_star.clone(),
+                        backup_address.to_string(),
+                    )));
+                }
+
+                for resp in responses {
+                    match resp.await
```

Comment:
This is okay for now, but since we only need M of N responses to return to make progress, eventually we'll probably want to move to using https://docs.rs/tokio/latest/tokio/task/struct.JoinSet.html, which is unfortunately part of tokio unstable (but at least doesn't require nightly rust : ) )